team.html

```diff
@@ -0,0 +1,378 @@
+							height="64"
+							viewBox="0 0 64 64"
+							fill="none"
+							xmlns="http://www.w3.org/2000/svg"
+						>
+							<title>Close pop-up window</title>
+							<path
+								d="M17.0668 50.6667L13.3335 46.9333L28.2668 32L13.3335 17.0667L17.0668 13.3333L32.0002 28.2667L46.9335 13.3333L50.6668 17.0667L35.7335 32L50.6668 46.9333L46.9335 50.6667L32.0002 35.7333L17.0668 50.6667Z"
+								fill="#1A1A1A"
+							/>
+						</svg>
+					</button>
+				</div>
+				<div class="modal-header">
+					<!-- marsha -->
+
+					<!-- marsha img -->
+					<div class="team-card-picture">
+						<picture>
+							<source
+								media="(min-width:768px)"
+								srcset="
+									./assets/images/pages/our-team/desktop/marsha-modal-d.jpg
+								"
+							/>
+							<img
+								src="./assets/images/pages/our-team/marsha.jpg"
+								alt="Dr. Marsha Maxwell, CETL"
+								style="width: auto"
+							/>
+						</picture>
+					</div>
+
+					<!-- Marsha info -->
+					<div class="team-card-info-modal">
+						<h2 class="h4-styles">Dr. Marsha Maxwell, CETL</h2>
+
+						<!-- div for icon and button -->
+
+						<!-- link svgs for email and linkedin -->
+						<div class="icons">
+							<h3 class="h5-styles">Co-founder</h3>
+							<a href="">
+								<!-- email icon -->
+								<img
+									src="assets/icons/our-team-icons/email-icon.svg"
+									alt="email icon"
+								/>
+							</a>
+							<a href="">
+								<!-- LinkedIn icon -->
+								<img
+									src="assets/icons/our-team-icons/linkedin.svg"
+									alt="linkedin icon"
+								/>
+							</a>
+						</div>
+					</div>
+				</div>
+				<div class="modal-content">
+					<div class="modal-scroll">
+						<p>
+							With strategic, tactical and operational technology experience,
+							Dr. Marsha Maxwell creates innovative environments and
+							applications that harness the power of emerging technologies and
+							neuroscience to enhance the educator's reach to better engage
+							students and foster a culture of innovation in organizations.
+						</p>
+						<p>
+							Her areas of interest include Augmented Analytics, Immersive
+							Experiences, Artificial Intelligence, Machine Learning, STEAM
+							programs, and cybersecurity. Marsha has also been very active in
+							bringing STEM/STEAM initiatives to life especially for underserved
+							and underrepresented groups throughout their educational journey
+							from kindergarten to university and beyond
+						</p>
+						<p>
+							Marsha has been a featured speaker for various organizations'
+							educational summits and workshops including Microsoft, Apple, and
+							Google for her skills in education technology, innovation
+							leadership, strategic analysis and decision-making, and equity and
+							inclusion in technology.
+						</p>
+					</div>
+				</div>
+			</div>
+		</section>
+
+		<div class="overlay-la hidden"></div>
+
+		<script src="/js/our-team-modal.js"></script>
```

Comment:
having some errors show up for the JS on my end. 

<img width="332" alt="Screenshot 2024-01-19 170008" src="https://github.com/indigenousfriends/itlct-website-redesign/assets/77703524/25f4941b-ecb5-4569-84fa-3a065d913809">

Reply:
we heard that this is no longer and issue right now, but if it continues we can fix it- just let us know